Update UG basic workflow section

HynRidge opened the pull request:
Note : Don't merge first!!

HynRidge committed:
Update UG basic workflow section

luo-git commented on docs/UserGuide.md:
```diff
@@ -28,10 +28,14 @@ will be able to manage and access your files with ease.
+3. To find the location of tagged files, use the `show` command to get the file path of the file.<br>
+![show](images/show.PNG)
+4. To exit the application, either close the application window, or use the `exit` command.<br>
+![Exit](images/exit.PNG)
```
I don't know if `show` and `exit` screenshots are necessary. Perhaps remove them?

Also maybe extra spacing will be nice.

HynRidge committed:
Merge branch 'master' into update-docs

# Conflicts:
#	docs/UserGuide.md

luo-git reviewed (approved):
Looks good except maybe we should change the screenshots to use dark theme.

HynRidge committed:
Merge branch 'master' into update-docs

HynRidge committed:
Update Illustrations

HynRidge committed:
Add warning to undo and redo command
Letting user know that undo and redo command can't be executed once app is restarted

HynRidge committed:
Merge branch 'master' into update-docs

HynRidge committed:
Delete images

HynRidge committed:
Add screenshots

luo-git commented on docs/UserGuide.md:
```diff
@@ -223,13 +225,19 @@ Format: `ls`
 Undo a recently executed command.<br>
 You can undo these commands: `tag`, `retag`, `untag`, `label`, `unlabel`, `clear`, and `redo`.
+<div markdown="block" class="alert alert-info">
+**Warning: Command history would be deleted once the app is close!
```
Grammatical error here: Command history will be deleted once the app is closed!

luo-git commented on docs/UserGuide.md:
```diff
@@ -223,13 +225,19 @@ Format: `ls`
 Redo a recently executed command.<br>
 Redo only exists when undo has been executed.
+<div markdown="block" class="alert alert-info">
+**Warning: Command history would be deleted once the app is close!
```
Same as above.

luo-git reviewed (changes requested):
Two grammatical errors.

HynRidge committed:
Fix gramatical error!

luo-git commented on docs/UserGuide.md:
```diff
@@ -226,7 +226,7 @@ Format: `ls`
 You can undo these commands: `tag`, `retag`, `untag`, `label`, `unlabel`, `clear`, and `redo`.
 <div markdown="block" class="alert alert-info">
-**Warning: Command history would be deleted once the app is close!
+**Warning: Command history will be deleted once the app is close!
```
Should be "is closed"?

luo-git commented on docs/UserGuide.md:
```diff
@@ -236,7 +236,7 @@ Format: `undo`
 Redo only exists when undo has been executed.
 <div markdown="block" class="alert alert-info">
-**Warning: Command history would be deleted once the app is close!
+**Warning: Command history will be deleted once the app is close!
```
Same as above.

luo-git reviewed (changes requested):
Oops you forgot to change the "close".

HynRidge committed:
Update UG!

luo-git reviewed (approved):
LGTM!
